OptimizelySDK/ProjectConfig.cs

```diff
@@ -353,5 +362,114 @@ public Variation GetVariationFromId(string experimentKey, string variationId)
             ErrorHandler.HandleError(new Exceptions.InvalidVariationException("Provided variation is not in datafile."));
             return new Variation();
         }
+
+        /// <summary>
+        /// Gets the forced variation for the given user and experiment.  
+        /// </summary>
+        /// <param name="experimentKey">The experiment key</param>
+        /// <param name="userId">The user ID</param>
+        /// <returns>Variation entity which the given user and experiment should be forced into.</returns>
+        public Variation GetForcedVariation(string experimentKey, string userId)
+        {
+            if (string.IsNullOrEmpty(userId))
+            {
+                Logger.Log(LogLevel.DEBUG, string.Format("User ID is invalid.", userId));
```

Comment:
no need for string format if you're not using `userId` in the string.

Reply:
done. removed string.format

Reply:
@msohailhussain 
Which branch are  you on? I can't seem to find the change on:

mng/ev_ns_fb

or 

phase2_ev_ns_fb_kr

Which branch should I be looking at?

Also, I can't find you on Slack with the user name msohail@folio3.com